rllib/tests/test_nested_observation_spaces.py

```diff
@@ -457,9 +457,9 @@ def test_multi_agent_complex_spaces(self):
                             PGTFPolicy, DICT_SPACE, act_space,
                             {"model": {"custom_model": "dict_spy"}}),
                     },
-                    "policy_mapping_fn": lambda a: {
+                    "policy_mapping_fn": lambda agent_id, **k: {
```

Comment:
Explicitly write out kwargs

Reply:
Removed the `*` and renamed it `**kwargs` in all tests/examples.